### PR TITLE
Feature/event details

### PR DESCRIPTION
**Pull request by hoddez**

### CODE REVIEW

File: sendgrid/constants.py [hunk 1]
```diff
@@ -6,15 +6,30 @@
...
+	"delivered":"DeliverredEvent",
+	"unknown":"Event",
+	"processed":"Event",
+	"open":"Event",
+	"unsubscribe":"Event",
+	"spamreport":"Event"
 }
+
+EVENT_TYPES_EXTRA_FIELDS_MAP = {
+	"UNKNOWN": (),
```
Comment (RyanBalfanz) [inline sendgrid/constants.py:24]: Should this be `None`?

Comment (hoddez) [inline sendgrid/constants.py:24]: nah, my test implementation assumes an iterable object so None breaks it. I suppose it should handle both None and empty list to be more robust

**Comment by RyanBalfanz** [comment]
:thumbsup:

**Comment by RyanBalfanz** [comment]
:shipit:

**Comment by RyanBalfanz** [comment]
See #15

**Pull request merged**